mautic/mautic
# Fix form mapped field radio and checkbox group company

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
I bet this issue was already fixed, maybe by me or @escopecz . But issue  now is back :(


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  Create select custom company (don't forget company object select) field with 3 options
2.  Create form and create radio and checkbox group with  enabled Use assigned contact/company field's list choices from properties tab.
3. See preview form and see form fields wasn't fill with options from custom fields

#### Steps to test this PR:
1.  Repeat all steps
2.  See  all form fields are pre-filled with custom company field options
